extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/RegistrationServiceExtension.java

```diff
@@ -14,25 +14,45 @@
 
 package org.eclipse.dataspaceconnector.registration;
 
+import org.eclipse.dataspaceconnector.extension.jersey.mapper.EdcApiExceptionMapper;
+import org.eclipse.dataspaceconnector.iam.did.spi.resolution.DidPublicKeyResolver;
 import org.eclipse.dataspaceconnector.registration.api.RegistrationApiController;
 import org.eclipse.dataspaceconnector.registration.api.RegistrationService;
+import org.eclipse.dataspaceconnector.registration.auth.DidJwtAuthenticationFilter;
 import org.eclipse.dataspaceconnector.registration.authority.DummyCredentialsVerifier;
 import org.eclipse.dataspaceconnector.registration.authority.spi.CredentialsVerifier;
 import org.eclipse.dataspaceconnector.registration.manager.ParticipantManager;
 import org.eclipse.dataspaceconnector.registration.store.InMemoryParticipantStore;
 import org.eclipse.dataspaceconnector.registration.store.spi.ParticipantStore;
+import org.eclipse.dataspaceconnector.spi.EdcSetting;
 import org.eclipse.dataspaceconnector.spi.WebService;
+import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
 import org.eclipse.dataspaceconnector.spi.system.ExecutorInstrumentation;
 import org.eclipse.dataspaceconnector.spi.system.Inject;
 import org.eclipse.dataspaceconnector.spi.system.Provider;
 import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
 import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
 
+import java.util.Objects;
+
+import static java.lang.String.format;
+
 /**
  * EDC extension to boot the services used by the Registration Service.
  */
 public class RegistrationServiceExtension implements ServiceExtension {
 
+    public static final String CONTEXT_ALIAS = "authority";
```

Comment:
would `registration` be more suitable?

Reply:
The API exposes more than just registration operations (it also allows enumerating dataspace participants).

Reply:
But the endpoint for enumerating participants is currently available at `/authority/registry`, so maybe we could shorten this to _just_ `/registry`?

On the other hand, if we leave `/authority`, we could have one single context path for all endpoints, even future ones. In that case I would rename the extension to something like `AuthorityExtension` or similar.

Reply:
Updated. Thanks.